src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1070,18 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString buildInfo = tr("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
+                           "<tr><td colspan=\"2\" style=\"font-style:italic;\">It would be useful to copy and include information below when reporting an issue!<br></td></tr>");
+
+    buildInfo.append(tr("<tr><td style=\"font-weight: 800\">Build information:</td><td>Build: %1</td></tr>").arg(mudlet::self()->version));
+    buildInfo.append(tr("<tr><td></td><td>OS: %1</td></tr>").arg(QSysInfo::prettyProductName()));
+    buildInfo.append(tr("<tr><td></td><td>CPU Architecture: %1</td></tr>").arg(QSysInfo::currentCpuArchitecture()));
+    buildInfo.append(tr("<tr><td></td><td>Branch: %1</td></tr>").arg(GIT_BRANCH));
+    QString commitHash = GIT_COMMIT_HASH;
+    buildInfo.append(tr("<tr><td></td><td>Commit: %1</td></tr>").arg(commitHash.mid(0, 8)));
```

Comment:
```suggestion
    buildInfo.append(tr("<tr><td></td><td>Commit: %1</td></tr>").arg(QByteArray(GIT_COMMIT_HASH).left(8)));
```
And loose the preceding line.